docs/en_us/platform_api/source/links.rst

```diff
@@ -1 +1,7 @@
-.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
\ No newline at end of file
+.. _OAuth 2.0 Standard: https://tools.ietf.org/html/draft-ietf-oauth-v2-31
+
+.. _edX platform documentation: http://docs.edx.org/
+
+.. _https://courses.edx.org/api/courses/v1/courses/: https://courses.edx.org/api/courses/v1/courses/
+
```

Comment:
ah, I thought it was just a comment because of the appearance here in light gray. I see that the formatting is correct. But, I'm pretty sure that a URL in an rst file is automatically linked to the destination by the Sphinx template (mail:to's get added automatically as well)